### DISCUSSION TITLE
Polarization and coherence properties of stellar radiation.

### DISCUSSION THREAD

**Discussion by carlosbelcech** [a-cosmology-group/acg#97] [Redshift Models]
Hello, I opening this post for to debate the properties of stellar radiation. I interested to know if there are measurements of the coherence degree of stellar sources and if exits some as "a polarization degree". I was studying the ponderomotive effect as a possible cause of redshift. I understand this effect such as is explained in the attachment. Two cuasi-colinear beams of coherent and plane polarized light (laser) with nearing frequencies can produces a interference pattern with a certain group velocity. This velocity can be arbitrarily slow, until to reach the thermal velocity of electrons of a hipotetic intergalactic plasma. If one pattern to coincide in velocity and direction with one electron, there are certain probability of produces the energy transference from higher energetic photon to the lowest one, as is explained in the attachment. Until now, I can get a redshift in base to this phenomenon, with a "number of events of transference" independent of the wavelength and this do not like me. 
I guess that this phenomenon could explain the z dependent  time dilation.
But, the question that I want to dilucide is if the stellar radiation could produce this patterns, still in a randomic way. I understand that it is necessary a certain degree of polarization and coherence.
[Astronomical redshift from ponderomotive effect.docx (1).pdf](https://github.com/a-cosmology-group/acg/files/11982488/Astronomical.redshift.from.ponderomotive.effect.docx.1.pdf)
[kozak2018.pdf](https://github.com/a-cosmology-group/acg/files/11982489/kozak2018.pdf)

**Comment by RedshiftDrift** [comment]
Stellar radiation is slightly polarized.  However, radiation emitted by matter in magnetic fields has a higher degree of polarization e.g. sun spots via Zeeman effect, cyclotron radiation via magnetic field, the CMB via "gravitational waves" (according to LCDM) or magnetic fields (according to a non-cosmological origin of the CMB).  Scattering off dust particles also polarizes radiation.  Longer coherence lengths are seen in specific objects like astronomical masers (methane, hydrogen, synchrotron) and Fast Radio Bursts, but other sources have a very short coherent length comparable to about a wavelength (as for any blackbody source).

Reading your notes and comments in the attached paper, you are about at the stage I was when I wrote ["Optical forces as a redshift mechanism: the “Spectral Transfer Redshift”"](http://adsabs.harvard.edu/abs/2009ASPC..413..268M)  I believe you are on the right track!

**Reply by RedshiftDrift** [reply]
@carlosbelcech, below are some comments on your proposal [Astronomical redshift from ponderomotive effect.docx (1).pdf](https://github.com/a-cosmology-group/acg/files/11982488/Astronomical.redshift.from.ponderomotive.effect.docx.1.pdf) [Belcech1] and on the notes you wrote on the paper by [Kozák *et al.*](https://github.com/a-cosmology-group/acg/files/11982489/kozak2018.pdf)  Except for a few minor things, you and I are in excellent agreement.

> The inelastic ponderomotive scattering is [an] interaction with a freepath dependent on wavelength. (from [Belcech1])
> I can get a redshift ... with a "number of events of transference" independent of the wavelength" (from [above](https://github.com/orgs/a-cosmology-group/discussions/97#discussion-5378292))

How do you eliminate the dependence on wavelength to obtain a cosmological redshift that is wavelength-independent?

> These [photons] are “replaced” by “substitute photons” generated by stimulated emission, with the energy and direction of the lower energy wave. [Belcech1]

This is exactly as I [described it](http://adsabs.harvard.edu/abs/2009ASPC..413..268M) *"One photon is transferred from one beam to the other by a stimulated emission process. ... As light is transferred into the lower energy beam every time there is an interaction, the mechanism produces a redshift of the radiation field."*

It is a strange feeling to realize that the light we receive from distant galaxies is entirely made up of *substituted photons*!

> In this frame it is possible to solve the Schrodinger’s equation for the electron target under the action of the two beams in the ponderomotive potential approximation. [Belcech1]

Did you actually solve the Schrödinger equation?  I used instead the density matrix formalism to calculate the "collision rate" for the dipole force on an atom (it's easier to account for the momentum transfer during a "collision") - we could compare our results which should be similar.

> "*... via stimulated Compton scattering. The strength of the interaction is proportional to the light intensity*" [Kozák *et al.*]
> Upa!, dice que es SCS y es lineal con I.

You're right, SCS is proportional to $I^2$ and cannot work for the cosmological redshift because intergalactic light is too weak.  The ponderomotive force ***is*** linear in $I$, which is a necessary condition for the mechanism to work.

> First order interactions are via Compton and Thomson scattering. The inelastic ponderomotive scattering is a second order interaction... [Belcech1]

The inelastic ponderomotive scattering is also a **first order** interaction, with a cross section that is $>10^6$ times the cross section of Compton scattering.  However, Kozák *et al.* write *"Albeit obtained in a second-order ponderomotive process, this gradient is already much higher than..."* because, for the experiment to give a detectable signal, they had to use high laser power.  The effect they observe is SCS, the second order manifestation of ponderomotive scattering.

> “*The second possible process (Fig. 1c) leads to a decrease of electron energy/momentum*” [Kozák *et al.*]
> claro, el electrón puede ser acelerado o frenado. a mi me importa lo 1ero. Se supone que para electrones a bajas energías debería ser lo más probable.

It turns out that the process where "a photon with higher energy is absorbed & a photon with lower energy is emitted" has the **same probability** as the process where "a photon with lower energy is absorbed & a photon with higher energy is emitted".  This is clear on Fig. 2a and 2b in Kozák *et al.*, since the electron energy spectrum is extended symmetrically toward higher and lower energies.  (The theory for this is explained by e.g. [Federov *et al.*](http://journals.aps.org/pre/abstract/10.1103/PhysRevE.55.1015)).

***However***, a quantum calculation that doesn't neglect momentum transfer to the electron shows that the process is asymmetric and in this case there is an net acceleration of the electrons.  But the interaction time must be long enough ($\gtrsim 10 \mu s$) without being disturbed by collisions (with other photons or other electrons).

> In a thermodynamic sense, the interaction between photons of the radiation field and the electrons transfers kinetic energy from the field to the cold plasma. [Belcech1]
> We propose to apply this formulation to the IGM conditions, where the electron kinetic energy is nearing to zero. [Belcech1]

Because of the cosmological redshift, the radiation field transfers its energy to the IGM (as you write in the first sentence).  Contrary to what the second sentence says, the IGM becomes very hot ($> 10^{10}$ K).

> “Therefore, the electron experiences a constant phase with respect to the light intensity modulation and is pushed out of the highintensity regions by the ponderomotive force (shown schematically in Fig. 1e).” [Kozák *et al.*]
> O sea, ¿hace falta que la velocidad de grupo del haz resultante coincida con la del electrón?. De esto modo el electrón "ve" una onda estacionaria y se produce el efecto del potencial ponderomotriz?

Yes, it is necessary for the group velocity of the interference to coincide (almost exactly) with that of the electron.  But since light in intergalactic space has a very broad spectrum, it is always possible to find a pair of light-waves with almost equal frequencies so that the electron "sees" a standing wave!

> ... the question that I want to dilucide is if the stellar radiation could produce this patterns, still in a randomic way. I understand that it is necessary a certain degree of polarization and coherence.

Since the energy-loss process is described by a probability (not a probability amplitude), the effects of different polarizations of the light-waves can simply be added together.  For the coherence, the electron "selects" pairs of waves that have the same phase, so effectively the electron does some filtering that increases the coherence by orders of magnitudes (from femtoseconds to microseconds).  The rejected photons will interact with another electron that happens to have the appropriate momentum and polarization.

**Reply by carlosbelcech** [reply]
Hello Louis, here my responses (very rudimentary):

"How do you eliminate the dependence on wavelength to obtain a cosmological redshift that is wavelength-independent?"
R: In the attached file (redshift.pdf).
"Did you actually solve the Schrödinger equation?"
R: I did not do it, There are a proposal for this question in the paper attached (Dellweg).
"However, a quantum calculation that doesn't neglect momentum transfer to the electron shows that the process is asymmetric and in this case there is an net acceleration of the electrons. But the interaction time must be long enough without being disturbed by collisions (with other photons or other electrons)."
R: This condition could be possible in the IGM?. This is supercool!
"Because of the cosmological redshift, the radiation field transfers its energy to the IGM (as you write in the first sentence). Contrary to what the second sentence says, the IGM becomes very hot"
R:Is this something verified?.  It seems to me that the net transfer of energy from the photon field to the electron field is very low per unit volume.
"Since the energy-loss process is described by a probability (not a probability amplitude), the effects of different polarizations of the light-waves can simply be added together. For the coherence, the electron "selects" pairs of waves that have the same phase, so effectively the electron does some filtering that increases the coherence by orders of magnitudes (from femtoseconds to microseconds). The rejected photons will interact with another electron that happens to have the appropriate momentum and polarization"
R:This seems some like a spontaneous coherence that is not intrinsic of the radiation?

[dellweg21.pdf](https://github.com/a-cosmology-group/acg/files/11994079/dellweg21.pdf)
[redshift.pdf](https://github.com/a-cosmology-group/acg/files/11994080/redshift.pdf)

**Reply by RedshiftDrift** [reply]
> "Did you solve the Schrödinger equation?" R: ... in the paper attached (Dellweg).

Dellweg uses a time-averaged ponderomotive potential $V(z) = \frac{e^2}{2mc^2} \langle A^2 \rangle_t$.  I've tried this approximation and it gives an average photon energy loss equal to **zero**.  The reason is that $V(z)$ is a conservative potential that doesn't change after the interaction photons-electron.

The ponderomotive potential must be described without the "time-average" approximation  This looks like

$$\hat{a}_1 e^{+i\vec{k}_1\cdot\hat{\vec{r}}} + \hat{a}_2 e^{+i\vec{k}_2\cdot\hat{\vec{r}}} + \hat{a}_1^\dagger e^{-i\vec{k}_1\cdot\hat{\vec{r}}} + \hat{a}_2^\dagger e^{-i\vec{k}_2\cdot\hat{\vec{r}}}$$

and accounts for changes of the potential when a photon is destroyed $\hat{a}$ and stimulated $\hat{a}^\dagger$ into the other beam.


> "... the IGM becomes very hot" R:Is this something verified?. It seems to me that the net transfer of energy from the photon field to the electron field is very low per unit volume.

The temperature of the IGM electrons is not measured, the density is too low.  However, we know the temperature of
- the sun's plasma $T_e = 10^6$ K,
- the Milky Way circumgalactic medium $T_e= 10^7$ K,
- galaxy clusters $T_e = 3\times 10^7$ K,
- hottest cluster i.e. Bullet cluster $T_e = 2\times 10^8$ K.

The IGM has a much lower density than in the examples above, therefore could have a hotter temperature.

The energy transfer *heating* the IGM is very low.  However, the energy transfer *cooling* the IGM can only result from electron-electron collisions via bremsstrahlung, which is even lower.  The electrons must collide frequently and at high velocity (i.e. be hot) to lose the energy acquired by the photon-redshift.

> "... the electron "selects" pairs of waves that have the same phase, so effectively the electron does some filtering that increases the coherence." R:This seems some like a spontaneous coherence that is not intrinsic of the radiation?

I haven't calculated that yet, but the electron is the first mediator between the *amplitudes* of the highly coherent waves at $\omega_1$ and $\omega_2$ to produce the energy-loss as a coherent effect.  Then all these interactions can be summed up to describe any coherence of the light waves.

> I don't like this result because ... the probability that a coincidence of velocity and direction occurs ... should also depend on this size, in addition to the intensity of the radiation and the density and velocity profile of the electrons. In other words, what I am describing gives an incorrect result.

The probability *that a coincidence of velocity and direction occurs* is small, but when the **electric fields of the waves** are intergated over all directions and over the entire distribution of electron velocities, the result is sufficiently large to account for the cosmological redshift.

Your initial idea is correct, it is only the equations you use that are not the correct ones.

**Reply by carlosbelcech** [reply]
"The ponderomotive potential must be described without the "time-average" approximation"
R: ok, I understand that this is a future task. You don't did it yet.

"The energy transfer heating the IGM is very low. However, the energy transfer cooling the IGM can only result from electron-electron collisions via bremsstrahlung, which is even lower. The electrons must collide frequently and at high velocity (i.e. be hot) to lose the energy acquired by the photon-redshift"
R: Ok, I understand that electrons don't has a way for to dissipate the heat. Or, its cooling is very poor. I agree. There are estimations of the magnetic field in the IGM. I do not remember the paper. But, I think that is possible to start of these estimations to calculate the thermal velocity of electrons. Faster electron velocities implies higher differences between $\omega_1$ and $\omega_2$, nothing else.

"Then all these interactions can be summed up to describe any coherence of the light waves" 
R: Sorry, but either I do not understand this affirmation or I do not agree. For me, there must be short-live patterns with which the electrons interact. You say that wave trains of one-wavelenght coherence longitude coincide spontaneous and randomicly to form longer patterns. Ok, I can agree.

"The probability that a coincidence of velocity and direction occurs is small, but when the electric fields of the waves are intergated over all directions and over the entire distribution of electron velocities, the result is sufficiently large to account for the cosmological redshift."
R: Yes, as there are a velocity distribution of electrons, the range of $\delta \omega= \omega_1 - \omega_2$ is wider. In my stone-age attempt I wanted to use average values. But, the range of angles must be restricted to the angular diameter of the sources. I think that the mixture of spectra of different profiles from different sources must be a secondary and almost negligible effect, on the contrary, this mixture  could to alterate the profiles, the specific "signatures"of the sources.

By another hand, the transfer delay from $\omega_1$ to $\omega_2$ can explain the z dependent time dilation.

One question: Do you have a best proposal for the group velocity equation?. I want to confirm this.

**Reply by RedshiftDrift** [reply]
> "...ponderomotive potential described without the "time-average"" R: ok, I understand that this is a future task. You don't did it yet.

I am working on it right now!

> "all these interactions can be summed up " R: Sorry, but either I do not understand this affirmation or I do not agree. For me, there must be short-live patterns with which the electrons interact. You say that wave trains of one-wavelenght coherence longitude coincide spontaneous and randomicly to form longer patterns.

Said differently:
- wave trains of one-wavelength (short) coherence are decomposed as a sum of waves with long-coherence (>100 meters),
- each long-coherence waves $\omega_i$ and $\omega_j$ interact to produce the redshift,
- all these interactions can be summed up to get redshift with short coherence light.

> One question: Do you have a best proposal for the group velocity equation?. I want to confirm this.

The group velocity is the same equation as the usual wave propagation in an electron plasma.

**Reply by carlosbelcech** [reply]
Ok, at this point I tell you that I was thinking at the moment of write the post in "Redshift models".

1) In relation to coherence degree of stellar radiation. I had searched some papers and had readed about the coherence properties of blackbody radiation, with a coherence length assigned of one wavelength. The generation of this radiation is attributed to independent oscillators that emit spontaneously with a quantified energy, where the stimulated emission is discarded. What happen if the stimulated emission is subvaluated?. We has speculated that in semiconductors the stimulated emission is present and can to produce appreciable effects in the optic properties. In the surface or in the atmosphere of Sun (chromosphere?), where are produced the photons that leave the star. There not stimulated emission?, Inclusive at high scale? In the stellar atmosphere (and surface), surely the atoms are highly ionized with its "electron populations" totally inverted, like in a Laser. The knock of a photon on an excited atom, Would not to produce a chain reaction of stimulated de-excitations in the surrounding gas?. These chain reactions, Would not generate something like swarms of coherent photons?. Is this a non-sense? Something like coherent wave fronts or coherent wave clusterings with determinate sizes, with its coherence length and coherence duration. The sunlight would is made with these flocks of coherent photons embedded and hidden in the whole of radiation. If this is so, the ponderomotive effect would conservate the coherence properties of radiation along its travel and I believe that would very difficult to detect these. The fraction "spontaneous" of radiation would be no significant. After all, we know that until free electrons can produce laser radiation. Perhaps, these hidden or non-discover properties of stellar radiation would be the unique initial assumption of the proposal that we discuss. If is verified later, would be a golazo.

2) In relation to ponderomotive effect equations, I was thinking in to approximate the excited state of free electrons in the electromagnetic field of the bichromatic pattern like a harmonic oscillator. Is to say, to consider to free electron as a confined particle in a potential well, the ground state is the free electron, when the electron to absorb a photon $h \nu$ and is excited, the oscillator step to an excited state, and later, when is knocked by the second photon $h (\nu +\delta \nu)$, return to the ground state with an energy plus $h (\delta \nu)$.

3) In relation to "coherence idea", the size of bichromatic patterns, as function of the coherence length of beams can be a (wavelength dependent) free parameter of future fits. Starting from Hubble's law or the value of H, would be possible to determine the coherence properties of radiation.
Sorry, I have that to tell you.

**Reply by RedshiftDrift** [reply]
Thanks for sharing this, I think I understand your idea.  A comment on what you write about the surface of the Sun (chromosphere).

You say "*surely the atoms are highly ionized with its "electron populations" totally inverted*" but inversion is **not possible** for the principal quantum levels of hydrogen or helium.  At best hydrogen masers emit coherent radio waves and helium can lase in the infrared.  But there is no laser levels in pure hydrogen or helium for visible light.  Sunlight is mostly blackbody radiation and its coherence length is very short (see [this paper](https://www.nature.com/articles/s41598-022-08693-0))

**Reply by carlosbelcech** [reply]
Yes. This paper is one of the firsts that the web shown when someone searches. Is one of that I saw.
I send you other.
Do you want to say that great part of the sunlight is from accelerated ions in the plasma?. bremsstrahlung? or accelerated charge radiation of isolated ions or electrons?
It's true, as you say, either helium nor hydrogen can lase in visible light, point for you. Must be another stuff.
But, don't matter, I hope that you can to advance.
PD: I go to study photoconductivity for now, before that they throw me out. I will back if I have something interesting. 

Carlos
[kastner95.pdf](https://github.com/a-cosmology-group/acg/files/12039658/kastner95.pdf)

**Comment by RedshiftDrift** [comment]
> Do you want to say that great part of the sunlight is from accelerated ions in the plasma?. bremsstrahlung? or accelerated charge radiation of isolated ions or electrons?

Yes, most of the light from the sun is produced by this incoherent process and appears as a blackbody spectrum.  The spectral lines listed in the `kastner95.pdf` paper are emitted/absorbed by elements present in very small quantity on the sun.

**Reply by carlosbelcech** [reply]
Ok, two point for you. Do you know about the Van Cittert-Zernike theorem? I'm exploring it, it could be the salvation. I think that if you reach a solid result you should try to publish it, for example, in "Galaxies". If I can outline a paper I'll keep you informed, although my chances are close to zero.

**Reply by RedshiftDrift** [reply]
See under "Assumptions of the theorem"

> [Quasi-monochromatic waves](https://en.wikipedia.org/wiki/Van_Cittert%E2%80%93Zernike_theorem#Quasi-monochromatic_waves)
> The van Cittert–Zernike theorem assumes that the source is quasi-monochromatic. ... This requirement implies that a radio astronomer *must restrict signals* through a ***bandpass filter***. Because radio telescopes almost always pass the signal through a relatively narrow bandpass filter, this assumption is typically satisfied in practice.

That's why earlier, I discussed ***filtering***
> "... the electron "selects" pairs of waves that have the same phase, so effectively the electron does some filtering that increases the coherence."

I'll post the news here when my paper is published.
